Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -5,7 +5,9 @@
 
 void SFilledGraphWidget::Construct(const FArguments& InArgs)
 {
-	InData2D = InArgs._InDataArray2D;
+	DataArrays = InArgs._DataArrays;
```

Comment:
As we get more and bigger DataArrays this copy could become quite time consuming. It's fine to leave it for now but it's worth thinking about ways to mitigate this as we might need them!